analysis/aws_rules_cis/aws_cloudtrail_modified.py

```diff
@@ -9,4 +9,4 @@
 
 
 def rule(event):
-    return event['eventName'] in CLOUDTRAIL_CHANGE_EVENTS
+    return event.get('eventName') in CLOUDTRAIL_CHANGE_EVENTS
```

Comment:
I think this is reasonable to do in general for all Python rules/policies, to be honest. Assume the keys may not exist